documentation/modules/deploying/con-hostname-verification-node-ports.adoc

```diff
@@ -6,7 +6,8 @@
 = Troubleshooting TLS hostname verification with node ports
 
 [role="_abstract"]
-Off-cluster access using node ports with TLS encryption enabled does not support TLS hostname verification. 
+Off-cluster access using node ports with TLS encryption enabled does not support TLS hostname verification.
+The reason is that Strimzi does not know what will be the address of the node where the broker Pod will be scheduled and cannot add it to the broker certificate.
```

Comment:
```suggestion
This is because Strimzi does not know the address of the node where the broker pod is scheduled and cannot include it in the broker certificate.
```